File: official/docs/java/events/RetrieveEvent.java
import com.easypost.EasyPost;

public class RetrieveEvent {
    public static void main(String[] args) {
        EasyPost.apiKey = System.getenv("EASYPOST_API_KEY");
        
        Event event = Event.retrieve("evt_...");

        System.out.println(event);
    }
}

File: official/docs/csharp/v3/events/retrieve-event.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using Newtonsoft.Json;
using EasyPost;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            EasyPost.ClientManager.SetCurrent("EASYPOST_API_KEY");

            Event @event = await Event.Retrieve("evt_...");

            Console.WriteLine(JsonConvert.SerializeObject(@event, Formatting.Indented));
        }
    }
}
